[2022-02-09 09:01:02,230] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:02,242] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:02,242] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:02,242] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:01:02,242] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:02,255] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2021-05-01 23:00:00+00:00
[2022-02-09 09:01:02,261] {standard_task_runner.py:52} INFO - Started process 474 to run task
[2022-02-09 09:01:02,267] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2021-05-01T23:00:00+00:00', '--job-id', '502', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmph12jicz7', '--error-file', '/tmp/tmpg1d4cqcl']
[2022-02-09 09:01:02,268] {standard_task_runner.py:77} INFO - Job 502: Subtask bigquery_external_table_task
[2022-02-09 09:01:02,342] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-05-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 09:01:02,389] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:01:02,413] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2021-05-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-01T23:00:00+00:00
[2022-02-09 09:01:02,415] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:01:03,013] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20210501T230000, start_date=20220209T090102, end_date=20220209T090103
[2022-02-09 09:01:03,042] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:01:03,085] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:16:52,387] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:16:52,406] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-05-01T23:00:00+00:00 [queued]>
[2022-02-09 09:16:52,406] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:16:52,406] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:16:52,406] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:16:52,427] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2021-05-01 23:00:00+00:00
[2022-02-09 09:16:52,435] {standard_task_runner.py:52} INFO - Started process 636 to run task
[2022-02-09 09:16:52,441] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2021-05-01T23:00:00+00:00', '--job-id', '606', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmprk9ategr', '--error-file', '/tmp/tmpmjh3ni03']
[2022-02-09 09:16:52,442] {standard_task_runner.py:77} INFO - Job 606: Subtask bigquery_external_table_task
[2022-02-09 09:16:52,516] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-05-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:16:52,566] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:16:52,589] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2021-05-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-01T23:00:00+00:00
[2022-02-09 09:16:52,591] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:16:54,078] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20210501T230000, start_date=20220209T091652, end_date=20220209T091654
[2022-02-09 09:16:54,142] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:16:54,188] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
